src/FSharpPlus/Lens.fs

```diff
@@ -116,10 +116,16 @@ module Lens =
     [<RequireQualifiedAccess>]
     module Map=
         let inline _item i f t = Map.InvokeOnInstance (fun x -> Map.add i x t) (f (Map.tryFind i t))
+        let inline _at i f t = Map.InvokeOnInstance
```

Comment:
I'm unsure. Perhaps it would be better to redefine `_item` instead of having two slightly similar functions?

Reply:
I agree, _at can do everything _item does and more when combined with non. But don't know how much you care about backwards compatibility... changing _item would produce a compile error everywhere it is used.  Also, the compile error is pretty non-helpful since it says something about obj can't be inferred.

Reply:
`_item` is not in the 1.0.0 release so it would probably be better to fix the signature of that function for `Map` and `ReadOnlyDictionary`

Reply:
Since otherwise there would be a need to fix the issue in a 2.0.0 release.